src/System.CodeDom/src/System/CodeDom/Compiler/CodeGenerator.cs

```diff
@@ -128,7 +131,10 @@ private void GenerateTypeMember(CodeTypeMember member, CodeTypeDeclaration decla
                 GenerateDirectives(member.StartDirectives);
             }
 
-            GenerateCommentStatements(member.Comments);
+            if (member.Comments.Count > 0)
```

Comment:
Why is this change necessary?

At this point in the lifecycle of System.CodeDom, we don't need to make changes like this to try to achieve some kind of artificial consistency.  In fact, we're more likely to break things than to fix things.

Reply:
Removed all count check